### DISCUSSION TITLE
Stun command works as Radar Flash rather than Flashbang

### DISCUSSION THREAD

**Discussion by Dexteriority** [winstxnhdw/lc-hax#384] [Q&A] [answered]
Is it possible to change how the /stun works?
Like instead of infinite flashbang, is it possible to make it works the same as Radar Booster Flash(ping) ??
Since some enemies does not get stun when using /stun, but radar booster flash does stun them.

**Comment by winstxnhdw** [comment, marked as answer]
I've added it in latest.

**Reply by Dexteriority** [reply]
You're the best!
